docker/Dockerfile

```diff
@@ -5,70 +5,40 @@ ENV \
     DEBIAN_FRONTEND=noninteractive \
     LANG=C.UTF-8
 
-RUN \
-    apt-get update --quiet && \
-    apt-get dist-upgrade --quiet --quiet && \
-    apt-get install --quiet --quiet --no-install-recommends \
-        ca-certificates \
-        libpq5 \
-        python2.7 \
-        wait-for-it && \
-    apt-get clean --quiet
+# Install locally from $ARTIFACT_DIR by default
+ARG debian_sources_entry="deb file:${ARTIFACT_DIR} ./"
 
-ARG debian_repo="https://uwcirg.jfrog.io/uwcirg/portal-deb"
+RUN echo $debian_sources_entry > /etc/apt/sources.list.d/truenth.list
```

Comment:
probably for legacy reasons, but why name the sources "truenth.list" when building the `portal` package?

Reply:
/etc/apt/sources.list.d/* entries must end in *.list to be read by apt/dselect.